pkg/mcs/resource_manager/server/manager.go

```diff
@@ -45,11 +47,12 @@ func NewManager(srv *server.Server) *Manager {
 // Init initializes the resource group manager.
 func (m *Manager) Init() {
 	handler := func(k, v string) {
-		var group ResourceGroup
-		if err := json.Unmarshal([]byte(v), &group); err != nil {
+		group := &rmpb.ResourceGroup{}
+		if err := proto.Unmarshal([]byte(v), group); err != nil {
+			log.Error("err", zap.Error(err), zap.String("k", k), zap.String("v", v))
 			panic(err)
 		}
-		m.groups[group.Name] = &group
+		m.groups[group.Name] = FromProtoResourceGroup(group)
 	}
 	m.storage().LoadResourceGroups(handler)
```

Comment:
for loading groups, do we need to change `loadRangeByPrefix(resourceGroupPath, f)` to `loadRangeByPrefix(resourceGroupPath+"/settings", f)`🤔